corehq/tabs/tabclasses.py

```diff
@@ -1584,6 +1571,18 @@ def sidebar_items(self):
                         },
                     ],
                 })
+        if self.couch_user.is_superuser:
+            from corehq.apps.accounting.models import BillingAccount
+            if toggles.DOMAIN_PERMISSIONS_MIRROR.enabled_for_request(self._request) \
```

Comment:
this toggle doesn't seem to be uniformly applied to the code that's impacted by this feature. Some places (like imports) check the toggle but other places don't. Is that something that needs to be addressed (maybe not in this PR)?

Reply:
Yeah, fair point, but I'm going to leave it for SaaS when they make this generally available, which should include removing the feature flag altogether.